### DISCUSSION TITLE
Force device state (bulb) when offline

### DISCUSSION THREAD

**Discussion by andrejkolesa** [Koenkk/zigbee2mqtt#6100] [Devices]
Hi,

is it possible to change device state of Philips bulb to OFF (so that HA shows bulb as OFF) even when device (bulb) is not powered.

Meaning: When someone turn off my wall switch, bulb loose power and is OFF, but HA still show as ON.
And since my wall switch is also z2m enabled, i send state for bulb to OFF  when wall display is turned off, but it still stays as ON in HA (in z2m logs i see that OFF was sent to bulb, but it was timeout).
Is it still possible to set state to OFF anyhow ?

Thank you and br,
Andrej

**Comment by clyra** [comment]
Hi,

I have a similar problem but after the power is restored. The point is that z2mqtt doesnt report back to HA the new state of a hue bulb after a power loss/restore. Looking at the logs, there's a "device_announce" when the bulb returns. We could build a automation to poll the bulb after receiving such message, but I would expect that z2mqtt itself did it. Are we missing something? a config parameter perhaps?

**Reply by jaimmor** [reply]
Having the same issue, have anybody found a fix or workaround to make this work?
